kernel/src/paging.c

```diff
@@ -4,6 +4,7 @@
 #define PAGETABLE_DEFAULT (PAGING_PRESENT | PAGING_WRITABLE)
 
 #define GET_TABLE_PTR(page_entry) (PageEntry*)(page_entry & PAGING_PTRMASK)
+#define GET_LAYER_PAGESIZE(depth) (1UL << (3 + 9 * depth))
```

Comment:
depth should probably be inclosed in parens because this is a macro (otherwise order of operation could do weird things). This is also true for many of the other macros.